SafeExamBrowser.SystemComponents/VirtualMachineDetector.cs

```diff
@@ -42,7 +42,7 @@ public bool IsVirtualMachine()
 
 			if (macAddress != null && macAddress.Count() > 2)
 			{
-				isVirtualMachine |= ((byte.Parse(macAddress[1].ToString(), NumberStyles.HexNumber) & 2) == 2 || macAddress.StartsWith("080027"));
+				isVirtualMachine |= (macAddress.StartsWith("525400") || macAddress.StartsWith("080027"));
```

Comment:
Could you make these strings constants and use an appropriate name? Currently, it is unclear what they mean and why a computer should be a virtual machine if the mac address starts with one of them...